app/scripts/modules/core/src/modal/modals.less

```diff
@@ -46,6 +46,9 @@
     flex-direction: column;
     flex: 1;
   }
+  .fill {
```

Comment:
why do we need this?

Reply:
I needed to remove `fill-flex` from the `pre`, because the ansi_to_html adds spans, and the flex was sorting all the spans in a column, effectively ruining the formatting.
But I needed to keep `flex: 1` because I wanted to fill the parent, so I had to make this class

Reply:
Thanks for the explanation! I'm a little afraid of having this style that only applies to one type of modal in the general modal stylesheet -- feels like it could easily get removed/changed without someone understanding that this is necessary only when we're formatting the modal contents with a particular library. Perhaps we could either add a comment with this explanation, or add a`JobManifestPodLogs`-specific stylesheet.

Reply:
Hmmm, what we really want here is for the element to fill the parent withoht being a flex container itself. Maybe we can call it "fill-no-flex"? Or I can put it as an inline style if that's allowed. Not sure if it's worth creating a different file just for this

Reply:
I'll go with `fill-no-flex`. It is general enough. I can imagine wanting an element to fill its parent, even if the element itself is not a flex box.

Reply:
sure!